examples/micro-ros_pub_sub/src/main.cpp

```diff
@@ -0,0 +1,107 @@
+#include <Arduino.h>
+#include <Wire.h>
+#include <micro_ros_platformio.h>
+#include <rcl/rcl.h>
+#include <rclc/rclc.h>
+#include <rclc/executor.h>
+#include <std_msgs/msg/int32.h>
+
+#if !defined(MICRO_ROS_TRANSPORT_ARDUINO_SERIAL)
+#error This example is only avaliable for Arduino framework with serial transport.
+#endif
+
+//publisher
+rcl_publisher_t publisher;
+std_msgs__msg__Int32 msg_pub;
+
+// subscriber
+rcl_subscription_t subscriber;
+std_msgs__msg__Int32 msg_sub;
+
+// publisher and subscriber common
+rcl_node_t node;
+rclc_support_t support;
+rcl_allocator_t allocator;
+rclc_executor_t executor;
+rcl_timer_t timer;
+unsigned int num_handles = 1 + 1;   // 1 subscriber, 1 publisher
+
+#define LED_PIN 13
```

Comment:
Examples should be as generic as posible, please leave out specific leds. 
You can use the [micro-ros_publisher example](https://github.com/micro-ROS/micro_ros_platformio/blob/main/examples/micro-ros_publisher/src/Main.cpp) as reference

Reply:
Actually, the LED_PIN is used in the original Arduino example. So it has a right to stay in, as this is a direct port from Arduini micro-ros examples.
I'm closing this pull-request nevertheless and will create a new one, sanitized with appropriate directory structure and all file, such as platformio.ini included.